styleguide/source/_patterns/subtitle.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Subtitle
```

Comment:
This file looks good but needs to be moved to the `01-atoms` directory so that it will appear in the Pattern info in PatternLab. (see attached screenshot)

![screen shot 2018-02-05 at 4 44 51 pm](https://user-images.githubusercontent.com/1653723/35832752-055a01ee-0a94-11e8-8fbf-7eacf05210c2.png)